oozie-to-airflow/templates/shell.tpl

```diff
@@ -15,6 +15,9 @@
  #}
 {{ task_id }} = bash_operator.BashOperator(
     task_id='{{ task_id }}',
-    bash_command="gcloud dataproc jobs submit pig --cluster={0} --region={1} --execute 'sh {{ bash_command }}'"
-            .format(PARAMS['dataproc_cluster'], PARAMS['gcp_region'])
+    bash_command="gcloud dataproc jobs submit pig --cluster={dataproc_cluster} --region={gcp_region} --execute \"{pig_command}\"".format(
+        dataproc_cluster=PARAMS['dataproc_cluster'],
+        gcp_region=PARAMS['gcp_region'],
+        bash_command="{{ pig_command }}"
```

Comment:
I think bash_command parameter is wrong here (should be pig_command).

Reply:
I updated a PR. Thanks.